<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>110</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Countdown</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="exitPushButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>60</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>10</y>
      <width>327</width>
      <height>23</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="timeHorizontalLayout">
     <item>
      <widget class="QSpinBox" name="hoursSpinBox"/>
     </item>
     <item>
      <widget class="QLabel" name="hoursLabel">
       <property name="text">
        <string>hours</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="minutesSpinBox">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="minutesLabel">
       <property name="text">
        <string>minutes</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="secondsSpinBox">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="secondsLabel">
       <property name="text">
        <string>seconds</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="startPushButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>40</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pausePushButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>40</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Pause</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopPushButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>60</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>43</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
